src/framework/preferences/css/stylus/SeparatedPanelPrefsEditorFrame.styl

```diff
@@ -211,3 +212,6 @@ html {
 
 // Theming
 build-themes-separatedPanelPrefsEditorFrame(themes);
+
+// Fonts
+build-fonts-prefs(fonts);
```

Comment:
@grrrero do we need this and the themes styles in this file, it seems that the generated CSS file is always paired with PrefsEditor.css which should already include it. Similar comment for SeparatedPanelPrefsEditor.styl. Could you check if we can remove those functions from these two files?